lib/yarn.lock

```diff
@@ -1637,6 +1637,14 @@ call-bind@^1.0.0:
     function-bind "^1.1.1"
     get-intrinsic "^1.0.0"
 
+call-bind@^1.0.2:
```

Comment:
The changes to this file don't seem like they're supposed to be here.

Reply:
> The changes to this file don't seem like they're supposed to be here.

This has been reverted. 
de30e03cac7495a06c44bbe9bef0c2ad520bdb71